lithops/standalone/backends/ibm_vpc/ibm_vpc.py

```diff
@@ -545,7 +545,7 @@ def _create_vpc_client(self):
         """
         Creates an IBM VPC python-sdk instance
         """
-        authenticator = IAMAuthenticator(self.iam_api_key)
+        authenticator = IAMAuthenticator(self.iam_api_key, url=self.config.get('iam_endpoint'))
```

Comment:
@kpavel what if it's not in config? since it's not mandatory..where is default value?

Reply:
Then get() will return None and IAMAuthenticator will use the default, hardcoded, iam endpoint url